packages/website/pages/terms.js

```diff
@@ -39,9 +40,9 @@ export default function TermsOfService() {
           If you do not agree to all the terms and conditions of this agreement,
           then you may not access the Website or use any services.
         </p>
-        <h2 className="chicagoflf">
```

Comment:
Just a comment, should probably run this page through axe dev tools to make sure the header tag hierarchy is consistent, i figure that's why they're being moved down to h3's but i wanted to make sure i put this in a comment for posterity.